src/main/java/org/opentripplanner/netex/loader/parser/TimeTableFrameParser.java

```diff
@@ -76,6 +76,7 @@ void setResultOnIndex(NetexEntityIndex netexIndex) {
     }
 
     private void parseJourneys(JourneysInFrame_RelStructure element) {
+        if(element == null) { return; }
```

Comment:
@assadriaz Do you now if this should e logged as a problem in the issue report, or are this just trying to map an optional reference - hence no check needed at the caller side. I am going to approve it "as is", but just wondered if you checked it.